lark/tree_templates.py

```diff
@@ -73,9 +76,9 @@ def _match_tree_template(self, template, tree):
             return res
 
 
-
 class _ReplaceVars(Transformer):
-    def __init__(self, conf, vars):
+    def __init__(self, conf: TemplateConf, vars: Mapping[str, Tree[str]]) -> None:
+        super().__init__()

```

Comment:
Since `Transformer` has an `__init__()`, it is best to call it in case it does something more important in the future.